language/src/main.rs

```diff
@@ -48,6 +48,7 @@ struct HacspecCallbacks {
     output_directory: Option<String>,
     output_type: Option<String>,
     target_directory: String,
+    version_control: Option<bool>
```

Comment:
This is an odd type...
How about usig an enum with `Init` and `Update` values instead?